.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on: [push, workflow_dispatch]
```

Comment:
workflow_dispatch is needed so we can start CI manually from command line, using github CI.

so we don't have to do that "dummy commit" thing to trigger CI for more tests to run